### PR TITLE
266/a11y aria labels

### PR DESCRIPTION
**Pull request by ColinBuyck**
# Pull Request Template

## Issue Overview

This PR addresses #266 

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Most of these changes were pretty straightforward. Just two things to note:
1) Opted for a more custom solution was around the Zoom in/Zoom out labeling. Since these buttons receive focus prior to the map itself, I added the group role on its container with the aria-label Map Controls so that users would know what the zoom relates to.

2) Changed the accordion approach a bit more significantly to mirror the approach here: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/

Once this is merged in, I will add it to the A11y Database as the first Doorway example 🖌️ 

## How Can This Be Tested/Reviewed?

These can be tested by pulling this down locally and using the built-in screen reader on each case.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: doorway-ui-components/src/text/Heading.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ export interface HeadingProps {
   className?: string
   priority?: number
   styleType?: HeaderType
+  id?: string
```
Comment (ColinBuyck) [inline doorway-ui-components/src/text/Heading.tsx:8]: Curious if there are any concerns with bringing this change into UI-C to allow for this functionality?

File: shared-helpers/src/locales/general.json
Comment (ColinBuyck) [inline shared-helpers/src/locales/general.json:1]: @sarahlazarich Do you want translations to go in for these aria-labels as part of this PR?

Comment (sarahlazarich) [reply shared-helpers/src/locales/general.json:1]: if possible! @kathyccheng do you think we'd be ok to translate these in google translate (or do you have localize access)? We just have three strings - Tentatively, “Zoom in”, “Zoom out”, “Map Controls” if no changes are requested in review.

Comment (sarahlazarich) [reply shared-helpers/src/locales/general.json:1]: @ColinBuyck, caught up with Kathy -- let's leave the translations off for now. We need to run them through localize and Izzie is the only one with access.

**Review by ludtkemorgan — APPROVED** [review]
💸

**Pull request merged**